authorize-idp-access-tokens/README.md

```diff
@@ -36,9 +41,7 @@ This implementation includes two artifacts:
     * curl
     * jq
     * npm
-5. Configure and obtain Client App credentials from the Identity Provider. These usually are client identifier and secret / password credential pairs.
-
-6. Obtain a JWT access token from the Identity Provider. The grant type and method to obtain the access token is specific to each IdP. However, it is common for Identity Providers to support the client_credentials grant type to issue access tokens to authorized applications.
+5. Obtain a Client ID and a JWT access token from an OpenID Identity Provider (you'll be using the values during the quickstart)
```

Comment:
Should it OpenID Provider. OpenID Identity Provider sounds redundant

Reply:
I've found multiple public reference using OpenID Connect Identity Provider and OpenID Identity Provider. I could use the former for completion sake.

Reply:
lets use the former